.gitignore

```diff
@@ -377,6 +377,7 @@ dmypy.json
 !.vscode/tasks.json
 !.vscode/launch.json
 !.vscode/extensions.json
+*.vscode
```

Comment:
I'd normally put editor warts into my user's global `.gitignore` so that my projects don't end up with everybody's different editor configurations in there, but it's not a big deal.

Reply:
I didn't know that there was a global .gitignore! Super, super useful, thanks for this